src/AirtableClient.php

```diff
@@ -31,7 +31,7 @@ public function findAll(string $table, ?string $view = null): array
             $view = '?view=' . $view;
         }
 
-        $url = $this->id .'/'. $table . $view;
+        $url = $this->airTableId .'/'. $table . $view;
```

Comment:
you may consider [urlencode instead](https://www.php.net/manual/fr/function.urlencode.php)

Reply:
Good idead Mickael ! Btw this code was rewritten in PR #1 with `sprintf()` but for some of the URLs, `urlencode()` could help a lot :)

Reply:
In this case I'd advice use of http_build_query as you can't introduce a broken uri with this function, but I'm fine with sprint() use too.